unlock-app/src/components/interface/pagination/Next.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { ArrowGroup, RightArrow } from './Pagination'
+
+const NextButtons = ({ numberOfPages, currentPage, goToPage }) => {
+  const goToNextPage = () => {
+    if (currentPage + 1 > numberOfPages) {
+      return
+    }
+    goToPage(currentPage + 1)
+  }
+  const goToLastPage = () => {
+    if (currentPage + 1 > numberOfPages) {
+      return
+    }
+    goToPage(numberOfPages)
+  }
+  return (
+    <ArrowGroup>
+      <RightArrow onClick={goToNextPage}>&gt;</RightArrow>
```

Comment:
I think it would be good to maybe have a `disabled` state when there are no next page?

Reply:
Not sure if that is possible. Because the button is a `div` and not a `button`. `div` doesn't take a `disabled` attribute.

Reply:
I understand but you could make its "style" show that it is disabled (lighter?) and with no click possible on it. I tend to think that if it looks the same but behave differently, this is not a great experience... 
I actually think it would be better to use `<a>` rather than div in order to remain semantic here?

Reply:
Makes sense. I'll make the changes.